src/smart-components/access-management/UserGroupsTable.tsx

```diff
@@ -162,25 +190,79 @@ const UserGroupsTable: React.FunctionComponent<UserGroupsTableProps> = ({
   );
 
   return (
-    <DataView ouiaId={ouiaId} selection={selection}>
-      <DataViewToolbar
-        ouiaId={`${ouiaId}-header-toolbar`}
-        bulkSelect={
-          <BulkSelect
-            isDataPaginated
-            pageCount={groups.length}
-            selectedCount={selected.length}
-            totalCount={totalCount}
-            pageSelected={pageSelected}
-            pagePartiallySelected={pagePartiallySelected}
-            onSelect={handleBulkSelect}
+    <Fragment>
+      {isDeleteModalOpen && (
+        <WarningModal
+          ouiaId={`${ouiaId}-remove-user-modal`}
+          isOpen={isDeleteModalOpen}
+          title={
+            <FormattedMessage
+              {...messages.deleteUserGroupModalTitle}
+              values={{
+                count: currentGroups.length,
+                plural: currentGroups.length > 1 ? intl.formatMessage(messages.groups) : intl.formatMessage(messages.group),
+              }}
+            />
+          }
```

Comment:
Looks like `title` props can only be `string`. You already have a message preconfigured to work with plural values
```suggestion
          title={intl.formatMessage(messages.deleteUserGroupModalTitle, { count: currentGroups.length })}
```